lib/appcanary/http.rb

```diff
@@ -23,6 +23,8 @@ def ship_gemfile(endpoint, config, &block)
     end
 
     def ship_file(endpoint, payload, config)
+      Appcanary.check_runtime_config!(endpoint, config)
```

Comment:
Let's keep shipfile "dumb" and assume if it gets this far the config is always valid. Perform this check in `update_monitor`? and maybe rename to `valid_monitor_config?`?